server/segment_rpc.go

```diff
@@ -66,37 +66,29 @@ func (h *lphttp) ServeSegment(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// oInfo will be non-nil if we need to send an updated net.OrchestratorInfo to the broadcaster
-	var oInfo *net.OrchestratorInfo
-
-	if paymentError := orch.ProcessPayment(payment, segData.ManifestID); paymentError != nil {
-
-		acceptableErr, ok := paymentError.(core.AcceptableError)
-		if !ok || !acceptableErr.Acceptable() {
-			glog.Errorf("Unacceptable error occured processing payment: %v", paymentError)
-			http.Error(w, paymentError.Error(), http.StatusBadRequest)
-			return
-		}
-		oInfo, err = orchestratorInfo(orch, sender, orch.ServiceURI().String())
-		if err != nil {
-			glog.Errorf("Error updating orchestrator info: %v", err)
-			http.Error(w, "Internal Server Error", http.StatusInternalServerError)
-			return
-		}
-
-		glog.Errorf("Acceptable error occured when processing payment: %v", paymentError)
+	if err := orch.ProcessPayment(payment, segData.ManifestID); err != nil {
+		glog.Errorf("error occured processing payment: %v", err)
```

Comment:
For brevity, we can just say "error processing payment"

Reply:
fixed in 0dbcf97df79a4ff27e77f4cc545ff7e9a11950cb